modules/server/main.tf

```diff
@@ -120,6 +120,7 @@ data "template_file" "forseti_server_startup_script" {
     forseti_version              = var.forseti_version
     forseti_server_conf_path     = local.server_conf_path
     forseti_home                 = var.forseti_home
+    forseti_enable_tracing       = var.forseti_enable_tracing
     cloudsql_proxy_arch          = var.cloudsql_proxy_arch
```

Comment:
@red2k18 Please remove line 123 here, as it's not needed in the startup script anymore